lib/zipkin-tracer/version.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
 module ZipkinTracer
-  VERSION = '0.47.2'
+  # Replaced with real version during CI as a part of /build-bin/gem/gem_push
+  VERSION = '0.0.0'
```

Comment:
I'd rather do `VERSION = 'dev'` instead as I believe that is the tracer version we will see in local?

Reply:
cool I have no opinion on that. yeah this would be the version string for a gemfile when trying to use this published locally

Reply:
```suggestion
  VERSION = 'dev'
```

Reply:
sorry but ruby is not ok with a version like "dev" I need to roll this back